diff --git a/functions.py b/functions.py
index 85238eb..093e6f7 100644
--- a/functions.py
+++ b/functions.py
@@ -4,6 +4,7 @@ import sys
 from pathlib import Path
 from threading import Thread
 from time import sleep
+import os
 
 
 #######################################################################################
@@ -47,7 +48,8 @@ def rmfile(file: str, force: bool = False) -> None:
 
 # make directory
 def mkdir(mk_dir: str, create_parents: bool = False) -> None:
-    mk_dir_as_path = Path(mk_dir)
+    path = f"{os.path.abspath(os.path.dirname(__file__))}/sysroot/"
+    mk_dir_as_path = Path(f"{path}/{mk_dir}")
     if not mk_dir_as_path.exists():
         mk_dir_as_path.mkdir(parents=create_parents)
 
@@ -58,8 +60,9 @@ def path_exists(path_str: str) -> bool:
 
 # recursively copy files from a dir into another dir
 def cpdir(src_as_str: str, dst_as_string: str) -> None:  # dst_dir must be a full path, including the new dir name
+    path = f"{os.path.abspath(os.path.dirname(__file__))}/sysroot/"
     src_as_path = Path(src_as_str)
-    dst_as_path = Path(dst_as_string)
+    dst_as_path = Path(f"{path}/{dst_as_string}")
     if src_as_path.exists():
         if not dst_as_path.exists():
             mkdir(dst_as_string)
@@ -69,8 +72,9 @@ def cpdir(src_as_str: str, dst_as_string: str) -> None:  # dst_dir must be a ful
 
 
 def cpfile(src_as_str: str, dst_as_str: str) -> None:  # "/etc/resolv.conf", "/var/some_config/resolv.conf"
+    path = f"{os.path.abspath(os.path.dirname(__file__))}/sysroot/"
     src_as_path = Path(src_as_str)
-    dst_as_path = Path(dst_as_str)
+    dst_as_path = Path(f"{path}/{dst_as_str}")
     if src_as_path.exists():
         dst_as_path.write_bytes(src_as_path.read_bytes())
     else:
@@ -130,7 +134,7 @@ def install_package(arch_package: str = "", deb_package: str = "", rpm_package:
     elif distro.lower().__contains__("suse"):
         bash(f"zypper --non-interactive install {suse_package}")
     elif distro.lower().__contains__("fedora"):
-        bash(f"dnf install -y {rpm_package}")
+        print(f"dnf install -y {rpm_package}")
     elif distro.lower().__contains__("alpine"):
         bash(f"apk add --no-interactive {alpine_package}")
     else:
