diff --git a/setup-audio b/setup-audio
index bf43641..64903db 100755
--- a/setup-audio
+++ b/setup-audio
@@ -59,8 +59,11 @@ if __name__ == "__main__":
     # This won't work for soundcards which don't currently exist, such as ones that need firmware installed
     # by this script, but those aren't known to have this issue
     # the ; true is there because that command will return a non-zero value for some reason
-    print_status("Initializing sound card")
-    bash("alsactl init; true")
+
+    # This however, does not work with rpm-ostree and should not be necessary.
+    # Alsa services already do a restore/init.
+    # print_status("Initializing sound card")
+    # bash("alsactl init; true")
 
     print_status("Audio setup finished! Reboot to complete setup.")
     print_status("If you still have any issues post-reboot, report them to https://github.com/WeirdTreeThing/chromebook-linux-audio")
