diff --git a/functions.py b/functions.py
index 4eea390..5a53f29 100644
--- a/functions.py
+++ b/functions.py
@@ -259,6 +259,7 @@ def mdn_config():
 def st_warning():
     print_warning("WARNING: Audio on AMD StoneyRidge Chromebooks will not work with the kernel provided by your distro as the audio driver requires a special kernel config that cannot be shipped in any distro. A custom kernel is required.")
     print_warning("You can get a prebuilt kernel from https://nightly.link/chrultrabook/stoney-kernel/workflows/build/main/stoney-kernel.zip")
+    exit(1)
 
 
 def avs_config(args):
