diff --git a/functions.py b/functions.py
index 747313d..18255f6 100644
--- a/functions.py
+++ b/functions.py
@@ -267,6 +267,8 @@ def st_warning():
     print_warning("WARNING: Audio on AMD StoneyRidge Chromebooks requires a patched kernel.")
     print_warning("You can get a prebuilt kernel for Debian/Ubuntu/Fedora from https://chrultrabook.sakamoto.pl/stoneyridge-kernel/")
     print_warning("For other distros, a patch file is included in that same link, under the patches directory.")

+    exit(1)
+
 
 def avs_config(args):
